include/mbedtls/des.h

```diff
@@ -3,7 +3,7 @@
  *
```

Comment:
We should also state this warning for enum constants or the like that indicate DES in high-level APIs. There are already warnings for `PSA_KEY_TYPE_DES`, but not for the DES enum constants in `cipher.h`. Please add warnings (at least a short statement, not necessarily a full-fledged `\warnings`) in `cipher.h`.